Coinbase converts (#2660)
rotki/rotki: docs/changelog.rst

* :feature:`2513` Users can now add/edit/delete all types of assets, not only ethereum tokens.
* :feature:`2424` Users will now see a progress bar while the automatic update is downloading, and proper notification messages in case of failure.
* :feature:`2515` Users will now be able to navigate back from the assets page using a button.

> * :feature:`1007` Users will see in their trade history their assets conversions from the coinbase exchange.

Reviewer: nitpick word reordering

```suggestion
* :feature:`1007` Coinbase exchange users will now be able to see asset conversions in their trade history.
```
